samples/CommunityToolkit.Maui.Sample/ViewModels/ViewControls/AvatarModel.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace CommunityToolkit.Maui.Sample.ViewModels.ViewControls;
+
+public class AvatarModel
+{
+	public Color? AvatarBackgroundColor { get; set; } = Colors.Black;
+	public FontAttributes? FontAttributes { get; set; }
+
```

Comment:
I understand this and appreciate it, however it is still used in Microsoft.MAUI and I believe it's an effective way to show the use of named font sizes.  Will make the change to a numeric font size if appropriate?

Reply:
I'd prefer not to add code that leverages an `[Obesolete]` API because it'll cause a breaking change in our Toolkit in a future release of .NET MAUI.

I'll ping the team to see what the replacement for `Device.GetNamedSize` should be 👍

Reply:
I have removed these and replaced with numeric values.